bundles/net.tourbook.device/src/net/tourbook/device/sporttracks/FitLogSAXHandler.java

```diff
@@ -374,15 +371,15 @@ public String getName() {
    }
    private class Lap {
 
-      private long startTime;
-      private long endTime;
+      private long _startTime;
+      private long _endTime;
    }
 
    private class Pause {
 
-      private long startTime;
-      private long endTime;
-      private long duration;
+      private long _startTime;
+      private long _endTime;
+      private long _duration;
    }
 
    public FitLogSAXHandler(final FitLogDeviceDataReader device,
```

Comment:
Frederic hast created a unit test which fails, I don't know how to handle this for a code which is not yet merged into the `main` branch

![mt-unit-test](https://user-images.githubusercontent.com/1283445/103750996-e4e2e400-5007-11eb-8421-e62558d5361d.png)

Reply:
maybe you can ask Frederic the reason of the failure(if it is legitimate or an error in  the test itself). I asked him 2 weeks ago to review the PR but I think he didn't have time yet to do that.

Reply:
> I asked him 2 weeks ago to review the PR but I think he didn't have time yet to do that.

Sorry, I may have missed one of your comment ?
Let me try to answer correctly:
When I designed the unit tests, I made them so that when they fail, the new file is written to disk so you can compare it easily with the expected one : https://github.com/wolfgang-ch/mytourbook/blob/546f93190ff1be038bb1bab05333accb06f04c3d/bundles/net.tourbook.tests/test/utils/Comparison.java#L81
When your tests fail, you should then see the file updated and that way you can see if it's an expected change or if the change was not supposed to happen.
When I look at your PR, I see that you modified TourData.java so I am thinking that it is likely "normal". What you need to do is update the unit tests so they succeed in an expected manner.
Here is an example with @wolfgang-ch recent [change](https://github.com/wolfgang-ch/mytourbook/commit/546f93190ff1be038bb1bab05333accb06f04c3d) in the TCX import that caused a change in the unit tests and that I updated in my subsequent [commit](https://github.com/wolfgang-ch/mytourbook/commit/3aa35808dd7af502e6ec4a760de3a0f704b8113b#diff-cda601a49c4a627685f971353b11fae271bc7c7895dd58cf21becd0a088c11ab)

![image](https://user-images.githubusercontent.com/25235665/103790759-de9c3a00-4ffe-11eb-943a-70323a6c1f42.png)


I hope that helps!

Reply:
@FJBDev  , hhmm you and @Frederic are the same person ?
anyway your comments help a lot thanks 👍  even though it can make my development speed slower :-).

Reply:
No, @frederic is another person that is not a MT contributor.
I'm glad it helps.!